src/main/java/uk/nhs/digital/nhsconnect/lab/results/model/edifact/DiagnosticReportDateIssued.java

```diff
@@ -0,0 +1,71 @@
+package uk.nhs.digital.nhsconnect.lab.results.model.edifact;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.NonNull;
+import uk.nhs.digital.nhsconnect.lab.results.model.edifact.message.EdifactValidationException;
+import uk.nhs.digital.nhsconnect.lab.results.model.edifact.message.Split;
+import uk.nhs.digital.nhsconnect.lab.results.utils.TimestampService;
+
+/**
+ * Example DTM+ISR:202001280957:203'
+ */
+@EqualsAndHashCode(callSuper = false)
+@Getter
+@AllArgsConstructor
+@Builder
+public class DiagnosticReportDateIssued extends Segment {
+
+    private static final String KEY = "DTM";
+    private static final String QUALIFIER = "ISR";
+    private static final String KEY_QUALIFIER = KEY + "+" + QUALIFIER;
```

Comment:
You've exposed `Segment.PLUS_SEPARATOR` so might as well make use of it
```suggestion
    private static final String KEY_QUALIFIER = KEY + PLUS_SEPARATOR + QUALIFIER;
```